### DISCUSSION TITLE
Practica1 - Abogabot (Finalizada)

### DISCUSSION THREAD

**Discussion by Chaksite** [Launch-X-Latam/MisionFrontEnd#261] [Show and tell]
Amigos, les comparto mi practica 1, recien la termine porque me ocupo en el trabajo y me ando poniendo al corriente.

Espero puedan darme su feedback, leo sus comentarios y sus sugerencias!

https://github.com/Chaksite/LaunchX-FrontEnd/blob/main/Practica1-Abogabot/README.md

**Comment by RIvanCF** [comment]
Quedó muy genial!! 
Me ayuda a mejorar y enriquecer todavía mas la primera práctica. 
Excelente contribución

**Comment by Rubensmorph** [comment]
!Wow! esta excelente tu trabajo, la verdad te quedó super bien, se ve que le dedicaste bastante tiempo a lo del IX y UI, los wireframes están geniales, gracias por compartir.
